packages/css-calc/test/additional/mod-rem-infinity.mjs

```diff
@@ -27,7 +27,7 @@ assert.strictEqual(
 );
 
 assert.strictEqual(
-	calc('mod(-0, infinity)'),
+	calc('mod(calc(-1 * 0), infinity)'),
```

Comment:
true negative zero can only be the outcome of a calculation, it can never be the result of parsing.